### PR TITLE
Cde create vis app mvp2

### PR DESCRIPTION
**Pull request by edlu77**
#579

Various updates to CDE create vis page

### CODE REVIEW

File: apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.html [hunk 1]
```diff
@@ -13,8 +13,7 @@ <h1 class="title">Create a Cell Distance Visualization</h1>
   <div class="card data-upload">
     <h2 class="header">
       <span class="step-number">1</span>
```
Comment (axdanbol) [inline apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.html:15]: Use new step indicator component from design-system

File: apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.theme.scss [hunk 2]
```diff
@@ -4,6 +4,7 @@
 @use '../../../styles/breakpoints' as breakpoints;
 
 $gray-palette: map.get(palettes.$palettes, 'gray');
```
Comment (axdanbol) [inline apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.theme.scss:6]: Move all the logic here into the regular scss file and use css variables instead

**Review by axdanbol — CHANGES_REQUESTED** [review]
Use css variables and components from the design system

**Comment by edlu77** [comment]
@LibbyUX Our design system has been implemented for the create vis page, please review and let me know if there are any changes needed!

**Review by LibbyUX — APPROVED** [review]
Amazing, thank you so much @edlu77!

I have more work to do on our text field inputs after playing with this preview, but that's a separate issue for the design system. I will expand text fields to have helper text for error states, absolute positioned below the input field.

Thank you for knocking this out, it looks great. Happy Friday!

**Review by axdanbol — APPROVED** [review]

**Pull request merged**